src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -3791,6 +3792,20 @@ QgsVectorLayerExporter::ExportError QgsPostgresProvider::createEmptyLayer( const
   {
     conn->PQexecNR( QStringLiteral( "BEGIN" ) );
 
+    // We want a valid schema name ...
+    if ( schemaName.isEmpty() )
```

Comment:
Ok, what I meant was `schemaName = schemaName.isEmpty() ? "current_schema" : quotedValue(schemaName);` here
and `schemaName` instead of `quotedValue(schemaName)` below.    Just one query less ;)

Reply:
I checked on psql and you cannot use current_schema to prefix a table. That's why I added an extra query to retrieve its value.

Reply:
you don't need to.  a not prefixed table lands in the current schema